help/de/Help_ch_Account-Actions.xml

```diff
@@ -450,7 +390,8 @@
 
     <para>Um ein neues Konto anzulegen wählen Sie in der Kontenübersicht in dem Menü
       <menuchoice>
-        <guimenu>Aktionen</guimenu><guimenuitem>Neues Konto...</guimenuitem>
+        <guimenu>A<accel>k</accel>tionen</guimenu><guimenuitem>Neues
+        <accel>K</accel>onto...</guimenuitem>
```

Comment:
`…`

Reply:
@fellen: was sollen mir dir drei Punkte sagen?

Reply:
Nach den [Gnome-Richtlinien](https://developer.gnome.org/hig/stable/typography.html.de) soll seit  Jahren die typographische horizontale Ellipse `…` statt ihrer Schreibmaschinenumschreibung `...`. verwendet werden

Reply:
ja, aber ich bin hier ja nicht im Quellcode. Und hier [https://github.com/Gnucash/gnucash-docs/pull/169#discussion_r643546770](url) habe ich dich so verstanden, dass "hellip" aus der Doku raus kann, weil drei Punkte besser zu lesen sind.

Reply:
Wenn du den Cursor mit den Pfeiltasten drüber schiebst, merkst du, daß es ein Zeichen mit  3 Punkten ist.

Reply:
und was bringt mir das in der Doku? Ich sehe noch nicht einmal einen Unterschied.
Mir ist das ja prinzipiell egal, mit "Finden und Ersetzen" ist das ja schnell im ganzen Text erledigt. Wie willst du das denn jetzt haben?

Reply:
und mit Lupe siehst du, daß die Ellipse enger gesetzt ist als 3 Einzelpunkte.

Am saubersten wäre wahrscheinlich ein separater Commit, der das in allen xml und po-Dateien im gesammten Repo ersetzt. Idealerweise mit anschließendem xmlformat.

Zur  Not reicht es aber, wenn du das machst, wenn du damit beginnst, eine Datei zu bearbeiten.t.

Reply:
ok, die drei Punkte habe ich mit "hellip" ersetzt.